client/src/components/DeliveryMethods.tsx

```diff
@@ -0,0 +1,30 @@
+import { ReactElement } from 'react';
+import { CustomizedChip } from '../utils/customized.components';
+import { Box } from '@mui/system';
+import Typography from '@mui/material/Typography';
+import { DeliveryMethod } from '../types/delivery-method.enum';
+import { Apps, Email, Message, Person, Phone } from '@mui/icons-material';
+import { Icon } from '@mui/material';
+
+type DeliveryMethodsProps = {
+    deliveryMethods: DeliveryMethod[];
+};
+
+const ICON_MAP = new Map([
+    [DeliveryMethod.IN_PERSON, <Person style={{ color: 'var(--theme-purple)' }} />],
```

Comment:
This reminds me that we'll need to do some refactoring with the styles eventually by defining a theme. Let's leave this as is for now